android/app/build.gradle

```diff
@@ -15,14 +15,44 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 def loadSigningConfiguration(String name) {
```

Comment:
I would have wanted to extract these few methods into their own file for cleanliness of this `build.gradle` but for some reason Gradle would stop picking them up even by adding the appropriate `apply from:`, so after a few tries I gave up. Room for improvement here, though.